packages/ui/tsup.config.ts

```diff
@@ -43,13 +43,13 @@ export default defineConfig(overrideOptions => {
     },
     dts: true,
     entry: {
-      button: 'src/button.tsx',
-      card: 'src/card.tsx',
-      connection: 'src/connection.tsx',
-      field: 'src/field.tsx',
-      input: 'src/input.tsx',
-      label: 'src/label.tsx',
-      seperator: 'src/seperator.tsx',
+      'primitives/button': 'src/primitives/button.tsx',
+      'primitives/card': 'src/primitives/card.tsx',
+      'primitives/connection': 'src/primitives/connection.tsx',
+      'primitives/field': 'src/primitives/field.tsx',
+      'primitives/seperator': 'src/primitives/seperator.tsx',
+      'components/sign-in/index': 'src/components/sign-in/index.tsx',
+      'components/sign-up/index': 'src/components/sign-up/index.tsx',
```

Comment:
Is the `/index` necessary here?

```suggestion
      'components/sign-in': 'src/components/sign-in/index.tsx',
      'components/sign-up': 'src/components/sign-up/index.tsx',
```